watertap/ui/fsapi.py

```diff
@@ -218,11 +220,29 @@ def validate_description(cls, v, values):
     @classmethod
     def validate_value(cls, v, values):
```

Comment:
If I read this right, we can allow users select an option from a list, or enter a string, int or float?

Reply:
That's true according to @MichaelPesce